rpm/build_rpms.bsh

```diff
@@ -152,27 +152,26 @@ popd
 
 #Prep the SOURCES dir for git-lfs
 echo "Zipping up current checkout of git-lfs..."
-  
-rm -rvf ${CURDIR}/tmptar
+
+echo "Cleaning ${CURDIR}/tmptar"
+rm -rf ${CURDIR}/tmptar
+
 mkdir -p ${CURDIR}/tmptar/git-lfs-${LFS_VERSION}
 pushd ${CURDIR}/..
   #I started running out of space in the docker, so I needed to copy a little less waste
   tar -c . --exclude tmptar --exclude repos | tar -x -C ${CURDIR}/tmptar/git-lfs-${LFS_VERSION}/
 popd
-pushd ${CURDIR}/tmptar/git-lfs-${LFS_VERSION}
-  git clean -xdf
```

Comment:
If I remember correctly the only reason the git clean was needed was to remove any untracked files that were in your working copy when running the scripts. So this is definitely NOT needed, just FYI